Makefile

```diff
@@ -58,14 +59,22 @@ gen-mocks: ## Generate mocks
 
 .PHONY: build
 build: ## Generate a binary in ./bin
+	@echo "==> Building binary"
 	go build -ldflags "${LINKER_FLAGS}" -o ${DESTINATION}
 
 .PHONY: install
 install: ## Install a binary in $GOPATH/bin
 	go install -ldflags "${LINKER_FLAGS}"
 
+.PHONY: gen-notices
+gen-notices: ## Generate 3rd party notices
+	@echo "==> Generating 3rd party notices"
+	@chmod -R 777 ./third_party_notices
+	@rm -Rf third_party_notices
+	go-licenses save "github.com/mongodb/mcli" --save_path=third_party_notices
+
 .PHONY: release
-release:
+release: gen-notices ## Use goreleaser to generate builds and publish
```

Comment:
That is a funky little go feature right there! 👏

Reply:
[q] Is there anyway to configure it to warn us if there is a potential commercial license that we're about to use that could cause us problems? Would be really cool if we got warning when we tried to pull them in, rather than when we audit the tool with legal later.

Reply:
Yes, if that's your kind of thing, https://github.com/google/go-licenses#checking-for-forbidden-licenses

Reply:
It's a @jamesbroadhead call, but my gut feels like it's something we should do if we can.

Reply:
@colm-quinn I created CLOUDP-56971 to follow on this just to keep this PR focused